easybuild/tools/module_generator.py

```diff
@@ -673,6 +674,9 @@ class ModuleGeneratorLua(ModuleGenerator):
     PATH_JOIN_TEMPLATE = 'pathJoin(root, "%s")'
     UPDATE_PATH_TEMPLATE = '%s_path("%s", %s)'
 
+    START_STR = '[==['
```

Comment:
make it a list instead of two items?

Reply:
Well, a tuple maybe, like this?

```python
START_END_STR = ('[==[', ']==]')
```

Not sure that's a lot better though, since that also means using `START_END_STR[0]` and `START_END_STR[1]` rather than the more straightforward `START_STR` and `END_STR`...